tools/InstallationTests/NetcoreTests/Common.ps1

```diff
@@ -360,7 +360,7 @@ Gets valid resource name
 #>
 function Get-ResourceName
 {
-  return "some-unique-rg-name"
+  return "some-unique-res-name"
```

Comment:
why not the same?

Reply:
```rs$(Get-RandomLetters)```

Reply:
There were some problems with making this random because some of the tests were not exactly "unit".  We are throwing all of these away once xUnit is ready, correct?